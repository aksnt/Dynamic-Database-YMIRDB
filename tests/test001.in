SET a 1 2
GET a
DEL a
GET a
BYE
