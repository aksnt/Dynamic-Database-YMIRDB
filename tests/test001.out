> ok

> [1 2]

> ok

> no such key

> bye
